process-compose/suzuka-full-node/process-compose.yml

```diff
@@ -75,3 +75,25 @@ processes:
       exec:
         command: curl http://0.0.0.0:30732
 
+  suzuka-indexer-processor:
+  
+    command : |
+      #/bin/bash
+      # todo: this fails with gRPC Unimplemented error on the RawData.GetTransactions call
```

Comment:
It would be nice to test the `indexer-processor` prior to proceeding. But, I think that's going to be difficult.